scipy/sparse/compressed.py

```diff
@@ -135,30 +140,32 @@ def check_format(self, full_check=True):
             basic check, O(1) operations (default True).
         """
         # use _swap to determine proper bounds
-        major_name,minor_name = self._swap(('row','column'))
-        major_dim,minor_dim = self._swap(self.shape)
+        major_name, minor_name = self._swap(('row', 'column'))
+        major_dim, minor_dim = self._swap(self.shape)
 
         # index arrays should have integer data types
         if self.indptr.dtype.kind != 'i':
-            warn("indptr array has non-integer dtype (%s)"
-                    % self.indptr.dtype.name)
+            warn("indptr array has non-integer dtype ({})"
+                 "".format(self.indptr.dtype.name), stacklevel=3)
         if self.indices.dtype.kind != 'i':
-            warn("indices array has non-integer dtype (%s)"
-                    % self.indices.dtype.name)
+            warn("indices array has non-integer dtype ({})"
+                 "".format(self.indices.dtype.name), stacklevel=3)
 
         idx_dtype = get_index_dtype((self.indptr, self.indices))
         self.indptr = np.asarray(self.indptr, dtype=idx_dtype)
         self.indices = np.asarray(self.indices, dtype=idx_dtype)
         self.data = to_native(self.data)
 
         # check array shapes
-        if self.data.ndim != 1 or self.indices.ndim != 1 or self.indptr.ndim != 1:
+        if any([self.data.ndim != 1,
+                self.indices.ndim != 1,
+                self.indptr.ndim != 1]):
```

Comment:
I'm -0 on this sort of change. I don't think it improves readability much.

Reply:
It's mostly to get the long line broken to <80 characters. I think both are not readable in that sense but I can also break the ifs instead.